### DISCUSSION TITLE
I want to get the percentage of the each letter(not only the output with the highest percentage)

### DISCUSSION THREAD

**Discussion by SongGeunil1** [snakers4/silero-models#22] [Q&A] [answered]
## ❓ Questions and Help
I am using silero models in torch very useful(Thank you for that!)

But I want to get the percentage of the each letter in each position.

In the code below, I think I can get what I want by using "**output**".

`test_files = glob('/gdrive/My Drive/Colab Notebooks/34668_20200113220653_0120.wav') 
batches = split_into_batches(test_files, batch_size=10)
input = prepare_model_input(read_batch(batches[0]),
                            device=device)

output = model(input)
for example in output:
    print(decoder(example.cpu()))`

But I don't know how to do it.
Can you please tell me how to do it?

**Comment by snakers4** [comment, marked as answer]
Hi, 

If I understand you correctly you can just use the model output as-is and / or write you own decoder.

The shape would be something like batch * frames * tokens. A list of tokens is in the json.

**Comment by SongGeunil1** [comment]
Thank YOU!
